src/scss/menulog.scss

```diff
@@ -1,90 +1,43 @@
 // =================================
-// Menulog Color palette
+// Menulog Color Scheme
```

Comment:
Can this file be a partial now?

Reply:
Do you mean, can it be in a folder called partials?

I could put it in a folder called themes, as that's kinda what it is I guess?

Reply:
It was more of a question really; as the filename doesn't start with an underscore it implies, to me, that this file would be used generate a separate CSS file. Is that correct?

Reply:
Ah ok – makes sense. Yeh, should be able to be a file partial now – will update now

Reply:
Just updated that now 👍